packages/tokens/config.js

```diff
@@ -4,31 +4,35 @@ const EDSStyleDictionary = StyleDictionary.extend({
   platforms: {
     scss: {
       transforms: [...StyleDictionary.transformGroup.scss, "name/kebabCase"],
-      buildPath: "dist/scss/",
+      buildPath: "dist/",
       files: [
         {
-          destination: "_variables.scss",
+          destination: "scss/_variables.scss",
```

Comment:
So now the platforms build to arbitrary sub folders because the commonality between them is the transforms rather than the files

Reply:
🤔 

I'm not that familiar with how these tokens are built/transformed. Does this mean that we should expect more sub-directories in dist?